Jellyfin.Api/Controllers/PackageController.cs

```diff
@@ -69,6 +73,7 @@ public PackageController(IInstallationManager installationManager, IServerConfig
         /// <returns>An <see cref="PackageInfo"/> containing available packages information.</returns>
         [HttpGet("Packages")]
         [ProducesResponseType(StatusCodes.Status200OK)]
+        [Produces(JsonDefaults.CamelCaseMediaType)]
```

Comment:
```suggestion
```

This is already added on the base controller

Reply:
> Don't get me wrong I do understand that this way not every client has to add images for the different status states, however your moq screenshot in the web repo shows that these status images would be overwritten by the plugin images anyway?
> IMHO a basic status text or 'waring-icon' with a hover status reveal to the left of the name in of the plugin would be a 'better' idea, as this would allow the the client to chose how to display the status instead of forcing an server sent image.
> 
> A very quick and dirty moq of what I mean (may be that this is for another PR, just note that I really am no fan of sending

Don't apologies; very happy for your comment - The icons were there more of a visual indicator for me whilst testing - (i'm crap at visual design). There are issues with them as can be seen with the web change. Personally i'm happy to rip out the statusImage after the this PR is fully tested - and leave it to the client.

This is one of the PR's that keeps growing.

Reply:
It is - but it doesn't seem to work. (Just tested again). The field names get passed with capitals.
Is it because there are multiple JsonDefaults?

Reply:
The produces is based on what the client requests. I don't think we should be adding a produces attribute to specific endpoints

Reply:
Part of this change gets rid of all the lowecase properties from two of the structures to meet the normal style (This got rid of quite a few warnings in the compiler)

The jellyfin-web expects these as lowercase properties - so not wanting to change anything - i made the output the same, which required this attribute.

Happy to put in a PR on jellyfin-web to correct this, but i didn't know what else it would break.

Reply:
Instead of adding the attribute to the controller, you should add the attribute to each property. Make sure to use the one from `System.Text.Json` and not `Newtonsoft.Json `